db/job.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS JOB_STATUS(
+    jstat_id    INTEGER CONSTRAINT pk_jstat PRIMARY KEY,
+    jstat_name  TEXT NOT NULL
+);
+
+INSERT INTO JOB_STATUS(jstat_id, jstat_name) VALUES
+    (0, 'NONE'),
+    (1, 'WAITING'),
+    (2, 'FINISHED');
+
+CREATE TABLE IF NOT EXISTS JOB(
+    job_id      BIGINT GENERATED ALWAYS AS IDENTITY,
+    job_email   TEXT NOT NULL,
+    job_image   INTEGER NOT NULL DEFAULT 0,
+    job_video   INTEGER NOT NULL DEFAULT 0,
+
+    CONSTRAINT pk_job PRIMARY KEY(job_id),
+    CONSTRAINT fk_job_image
+        FOREIGN KEY(job_image)
+        REFERENCES JOB_STATUS(jstat_id)
+        ON DELETE SET DEFAULT,
+    CONSTRAINT fk_job_video
+        FOREIGN KEY(job_video)
+        REFERENCES JOB_STATUS(jstat_id)
+        ON DELETE SET DEFAULT,
+    UNIQUE(job_email)
```

Comment:
Why does this need to be unique? So someone can't have two jobs running at the same time? Does that mean we are planning to delete rows from the job table once a job is completed?

Reply:
Its unlikely a user would intentionally submit multiple jobs at the same time, though this was primarily to prevent spamming/flooding, or accidental resubmissions. These can be better controlled for on the JS side if we add submission and completed time information.

The previous jobs should be deleted, as I don't see a good reason to keep the emails of prior users. We would also have to keep at least the images for the jobs result to be useful, so would run in to storage limitations. When (if) we add accounts keeping a record (but not the data) of prior jobs would be valid.